src/screens/FilterTransactionsScreen.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useState } from 'react';
+import { View, StyleSheet } from 'react-native';
+import { DateObject } from 'react-native-calendars';
+import { NavigationScreenProps } from 'react-navigation';
+
+import { Header, ScreenTemplate, InputItem } from 'app/components';
+import { Button } from 'app/components/Button';
+import { Calendar } from 'app/components/Calendar';
+import { CardGroup } from 'app/components/CardGroup';
+import { RowTemplate } from 'app/components/RowTemplate';
+import { Route } from 'app/consts';
+import { processAddressData } from 'app/helpers/DataProcessing';
+
+const i18n = require('../../loc');
+
+enum Index {
+  From = 0,
+  To = 1,
+}
+
+export const FilterTransactionsScreen = (props: NavigationScreenProps) => {
+  const [address, setAddress] = useState('');
+  const [dateKey, setDateKey] = useState(0);
+  const [isCalendarVisible, setIsCalendarVisible] = useState(false);
+  const [fromDate, setFromDate] = useState('');
+  const [toDate, setToDate] = useState('');
+  const [fromAmount, setFromAmount] = useState('');
+  const [toAmount, setToAmount] = useState('');
+
+  const onFilterButtonPress = () => {};
+
+  const onDateSelect = (date: DateObject) => {
+    setIsCalendarVisible(false);
+    switch (dateKey) {
+      case Index.From:
+        return setFromDate(date.dateString);
+      case Index.To:
+        return setToDate(date.dateString);
+    }
+  };
+
+  const showCalendar = (index: number) => {
+    setIsCalendarVisible(true);
+    setDateKey(index);
+  };
+
+  const renderCommonCardContent = () => (
+    <>
+      <View style={styles.spacing10}>
+        <RowTemplate
+          items={[
+            <InputItem
```

Comment:
On input focus the smartphone keyboard is shown (after 1 sec it's hiding). We need to prevent keyboard shown:
https://stackoverflow.com/questions/45323023/i-want-keyboard-not-to-show-up-at-all-when-i-touch-my-text-input-react-native

Reply:
Also we need it for the address picker

Reply:
Let's add `<TouchableX>` on top of input and it should be okay